plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -253,6 +253,15 @@ private function get_feature_definitions() {
 					),
 					'enabled_by_default' => true,
 					'disable_ui'         => true,
+
+					/*
+					 * This is not truly a legacy feature (it is not a feature that pre-dates the FeaturesController),
+					 * but we wish to handle compatibility checking in a similar fashion to legacy features. The
+					 * rational for setting legacy to true is therefore similar to that of the 'order_attribution'
+					 * feature.
+					 *
+					 * @see https://github.com/woocommerce/woocommerce/pull/39701#discussion_r1376976959
+					 */
 					'is_legacy'          => false,
```

Comment:
We should set this to true, right?

```suggestion
					'is_legacy'          => true,
```

Reply:
🤦🏼 

Yes! 

(I was trying out different testing instructions before I pushed the changes, and had flipped the switch back to simulate behaviors *without* this change ... sorry.)